lib/hope-tab-list.js

```diff
@@ -27,6 +27,13 @@ fxosComponent.register('hope-tab-list', {
 
     // Events
     scheduler.attachDirect(this.els.inner, 'scroll', this.onScroll);
+    this.els.inner.addEventListener('scrollend', (evt) => {
+      // We detach the direct block to flush the scheduler queue then re-attach it
+      scheduler.detachDirect(this.els.inner, 'scroll', this.onScroll);
+      setTimeout(() => {
+        scheduler.attachDirect(this.els.inner, 'scroll', this.onScroll);
+      });
+    });
```

Comment:
I don't quite understand this code, looks a little hacky? When you say 'flush the queue' what tasks are you referring to? SHould there be an API for this?